test/data/snapshots/README.md

```diff
@@ -1 +0,0 @@
-This directory does not contain actual data, but it should be left in place. It is here to support testing.
```

Comment:
I understand that this file is not necessary anymore due to the way `initialize` now works, creating the folder hierarchy as needed, whereas it used to be mandatory for ensuring presence of the folders.
If that is indeed the case, then do we still need to mention those files in `data/README.md`? Do we still need the `test/data/README.md` altogether?